CHANGELOG.md

```diff
@@ -61,8 +61,9 @@ You can find its changes [documented below](#070---2021-01-01).
 - `scroll_to_view` and `scroll_area_to_view` methods on `UpdateCtx`, `LifecycleCtx` and `EventCtx` ([#1976] by [@xarvic])
 - `Notification::route` ([#1978] by [@xarvic])
 - Build on OpenBSD ([#1993] by [@klemensn])
-- Scope: expose scoped state using state() and state_mut() ([#2082] by [@rjwittams]
-- Tabs: allow getting and setting the tab index of a Tabs widget ([#2082] by [@rjwittams]
+- Scope: expose scoped state using state() and state_mut() ([#2082] by [@rjwittams])
+- Tabs: allow getting and setting the tab index of a Tabs widget ([#2082] by [@rjwittams])
+- App: add start_console_logging(bool) which accepts a flag. ([#2102] by [@ratmice])
```

Comment:
Unfortunately the links aren't automatically generated. Please manually define both `[#2102]` and `[@ratmice]` at the bottom of the file.

Reply:
Rebased and defined the links, sorry I had missed your comment until now.